x-pack/platform/plugins/shared/lens/public/react_embeddable/lens_embeddable.tsx

```diff
@@ -59,6 +59,14 @@ export const createLensEmbeddableFactory = (
     buildEmbeddable: async ({ initialState, finalizeApi, parentApi, uuid }) => {
       const titleManager = initializeTitleManager(initialState.rawState);
 
+      titleManager.api.title$?.subscribe((newTitle) => {
```

Comment:
If I'm correct [here](https://github.com/elastic/kibana/pull/225664/files#r2175607853) we should not need to subscribe to changes or mutate the `initialState.rawState.title`.